index.bs

```diff
@@ -2006,7 +2007,7 @@ been used for the attestation=] is consistent with the fields of the attestation
     Request a Android Key Attestation by calling "keyStore.getCertificateChain(myKeyUUID)") providing |attToBeSigned| as the
     challenge value (e.g., by using
     <a href="https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.Builder.html#setAttestationChallenge(byte%5B%5D)">
-    setAttestationChallenge</a>),
+    setAttestationChallenge=]),
```

Comment:
should be fixed as of commit https://github.com/w3c/webauthn/pull/371/commits/eee29f9e4d11c7038c1f0a0ba603d6f7e3b76cc5